docker/docker-compose.yml

```diff
@@ -1,14 +1,130 @@
 version: "3.5"
 
+x-environment:
+  &environment
+  PYTHONDONTWRITEBYTECODE: 1
+  PYTHONUNBUFFERED: 1
```

Comment:
Are `PYTHONDONTWRITEBYTECODE` and `PYTHONUNBUFFERED` needed? They're also defined in `functionary.Dockerfile`.

Reply:
Not really.  It certainly wasn't hurting anything, but I removed them anyway.